daprdocs/content/en/developing-applications/building-blocks/actors/namespaced-actors.md

```diff
@@ -0,0 +1,57 @@
+---
+type: docs
+title: "Namespaced actors"
+linkTitle: "Namespaced actors"
+weight: 40
+description: "Learn about the multi-tenant placement service in actors"
+---
+
+Namespaced actors use the multi-tenant placement service. For example, sidecars belonging to "Tenant A" don't receive placement information meant for "Tenant B". 
+
+<!-- need diagram --> 
+
+In order for multiple tenants to have actor types and/or IDs with the same name, every namespace should have its own state store. Otherwise, apps in different namespaces with the same actor type and/or ID may overwrite each other's data in the state store.
+
+## Example
+
+In the following example...
+
+<!-- need example? ok to use from placement API? --> 
+
+## Migrating data to a new namespace
+
+If you're moving to a new namespace and starting to use a new state store, make sure you migrate your data.
+
+## Backwards compatibilty
+
+Namespaced actors are backwards compatible, allowing you to block newer sidecar versions from seeing the actor types within older sidecar versions.
```

Comment:
Namespaced actors are backwards compatible for deployments that use mTLS,  because the sidecar's namespace is inferred from the Spiffe ID, allowing for multi-tenancy out of the box.
When mTLS is not enabled we default to what is explained below.